File: AMS_资讯信息组件/code/client/source/YssSecInformation.Support/Sv/Service/ISecBasePjService.cs
﻿using FAST.Core.Util;
using FAST.Common.Service.Pojo;
using FAST.Common.Service.Services.Base;
using FAST.Core.Communication.Service;
using FAST.Platform.Workflow.Pojo;
using FAST.Common.Service.Pojo.Base;
using FAST.Core.Exceptions;

using FAST.Common.Service.Attributes;
using FAST.Core.Context;

using System.Collections.Generic;

namespace YssSecInformation.Support.Sv.Service
{
    /// <summary>
    /// 票据基本信息数据服务接口
    /// 
    /// Create By Tangshifeng 2013-03-14
    /// </summary>
    [NewServiceAttribute(ServiceType.NEW_SVC)]
    [ServiceAttribute(ServiceId = YssInformation.Support.common.ServiceIdConstant.OSGI_BASEBUSINESS, ServiceController = "com.yss.ams.sec.information.support.modules.sv.base.controller.ISecBasePjController", ServiceUrl = "")]
    public interface ISecBasePjService : IServiceBus
    {
        /// <summary>
        /// 条件查询
        /// </summary>
        /// <param name="quyCon">1</param>
        /// <returns>查询结果对象</returns>
        [MethodAttribute(Method = RequestType.POST, MethodUrl = "/selectByCondition")]
        QueryRes selectByCondition(string quyCon);
      
    }
}

File: AMS_资讯信息组件/code/server/source/com.yss.ams.sec.information.support/src/main/java/com/yss/ams/sec/information/support/modules/sv/base/service/ISecBasePjService.java
package com.yss.ams.sec.information.support.modules.sv.base.service;

import com.yss.framework.api.common.co.BasePojo;
import com.yss.framework.api.mvc.biz.IServiceBus;
import com.yss.framework.api.restful.annotations.RestfulSupported;

/**
 * 票据基本信息 2013-03-14
 * @author Tangshifeng
 *
 */
@RestfulSupported
public interface ISecBasePjService extends IServiceBus {

	public BasePojo queryPjSecBase(String secCode);
}
